README.md

```diff
@@ -373,16 +385,21 @@ Otherwise, they will be resent at the next interval.
     >>
     >> [
     >>   {
-    >>     topic: <name>,
-    >>     type:  <type>,
-    >>     url:   <url>,
-    >>     t:     <t>
+    >>     topic: <string>,
+    >>     type:  <string>,
+    >>     url:   <string>,
+    >>     t:     <integer>,
+    >>     data:  <anything>
```

Comment:
The internal code requires data to be a `Hash` though, which is why I pointed it out. These docs are more lax than the actual implementation.